tests/conftest.py

```diff
@@ -360,17 +337,12 @@ def factory(
             project = project or active_project or project_factory()
             active_project = project
             if interface == constants.CEPHBLOCKPOOL:
-                storageclass = (
-                    storageclass or active_rbd_storageclass
-                    or storageclass_factory(interface)
-                )
+                storageclass = helpers.default_storage_class(interface_type=interface)
```

Comment:
Good to give some room for those tests which need to create new storage class. By adding a new parameter 'use_default_sc' which will be set to True by default..

Reply:
I think the whole point is to use defaults, there is still old storageclass factory for that for someone who needs that and can call that explicitly.

Reply:
Okay, That make sense.
So, the storage class has to be passed from test as parameter storageclass. That needs to be considered here.
```suggestion
                storageclass = storageclass or helpers.default_storage_class(interface_type=interface)
```

Reply:
Talked to @jilju, will revisit this later when needed.

Reply:
For some reason, I can't see this fix in the latest commit

Reply:
it is important not to break the option to create a new SC/pool/secret